experiments/ClimaEarth/run_amip.jl

```diff
@@ -855,13 +882,12 @@ if ClimaComms.iamroot(comms_ctx)
 
             # define variable names and output directories for each diagnostic
             amip_short_names_atmos = ["ta", "ua", "hus", "clw", "pr", "ts", "toa_fluxes_net"]
-            output_dir_atmos = atmos_sim.integrator.p.output_dir
             amip_short_names_coupler = ["F_turb_energy"]
             output_dir_coupler = dir_paths.output
 
             # Check if all output variables are available in the specified directories
-            make_diagnostics_plots(
-                output_dir_atmos,
+            make_ci_plots(
```

Comment:
```suggestion
            make_diagnostics_plots(
```
this was renamed recently